frontend/src/shared/modules/saved-views/services/saved-views.service.ts

```diff
@@ -0,0 +1,61 @@
+import authAxios from '@/shared/axios/auth-axios';
+import AuthCurrentTenant from '@/modules/auth/auth-current-tenant';
+import { SavedViewCreate } from '@/shared/modules/saved-views/types/SavedViewsConfig';
+
+export class SavedViewsService {
```

Comment:
When we merge this with crowd-linux I believe we will need to remove segments from requests for this service

Reply:
you mean we will need to add segments?